packages/next/build/entries.ts

```diff
@@ -101,6 +103,63 @@ type Entrypoints = {
   edgeServer: webpack5.EntryObject
 }
 
+export async function getPageRuntime(
+  pageFilePath: string,
+  globalRuntime?: string
+) {
+  let pageRuntime: string | undefined = undefined
+  const pageContent = await fs.promises.readFile(pageFilePath, {
+    encoding: 'utf8',
+  })
+  // branch prunes for entry page without runtime option
+  if (pageContent.includes('export const config =')) {
```

Comment:
We should only filter based on the `"runtime"` keyword, I don't think other conditions are reliable. For example people can do `export const config=` (w/o the space before `=`) and this will break.